Misc/NEWS.d/next/Library/2024-08-22-09-37-48.gh-issue-123213.owmXnP.rst

```diff
@@ -0,0 +1,3 @@
+:meth:`xml.etree.ElementTree.Element.extend` and Element assignment no
```

Comment:
"Element assignment" should either link to the corresponding method in the docs, or `Element` should be linked.